package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I'm guessing that you upgraded your version of node/npm, and now it's creating "package-lock.json" with `lockfileVersion: 2`...

This is causing a huge diff for that file, which means we will all also have to update node/npm so that every other PR won't have this huge diff (version 1 to 2 differences).

We could all update, but last time we tried that, the GHA didn't work with the later version. (I can't remember if that was 1 project only or all of them.) Anyway, we had a good reason to not update last time.

It's possible the GHA will work with the new node/npm version now, but we can't tell until GHA is fixed.

So, let's keep an eye on this. If it's OK with this project then we will need to test all other projects. If that's OK then we can update all projects.

(I'd prefer to not require a npm version manager since I jump between project a lot.)

Or, we could defer this node/npm update until a future ticket. Thoughts?

Reply:
Yes, thanks. My nvm wasn't set to have node 14.15.1 as default. I fixed that and rerun `npm i`. It'll be fixed in the next commit.